ibmcloud/README.md

```diff
@@ -89,9 +89,16 @@ As usual, you need to create `terraform.tfvars` to specify parameter values. The
 
 ```
 ibmcloud_api_key = "<your API key>"
-ssh_key_name = "<your SSH key name>"
+user_ssh_key_name = "<your SSH key name>"
```

Comment:
That's just a miss on my part to update line 112.

Reply:
I think this variable could keep the original name. 
The only thing to be careful of is to avoid confusion between `var.ssh_key_name` and `local.ssh_key_name` in the code. Perhaps change the local variable name instead so that the external name is shorter?